ros_buildfarm/templates/devel/devel_task.Dockerfile.em

```diff
@@ -87,7 +87,12 @@ RUN python3 -u /tmp/wrapper_scripts/apt.py update-install-clean -q -y ccache
 ))@
 
 @[if run_abichecker]@
-RUN python3 -u /tmp/wrapper_scripts/apt.py update-install-clean -q -y abi-compliance-checker python3 python3-catkin-pkg-modules python3-pip 
+# Focal abi-compliance-checker package has a bug that breaks python invocation
```

Comment:
Shouldn't the logic be conditional on `focal` then?

Reply:
Yes 8441c39

Reply:
If the same problem applies to newer Ubuntu distros the condition should be changed to ensure this workaround is being used in the future (until a fix has landed upstream and it can be removed).

Reply:
The same patch is shipped in Ubuntu Groovy. Changed in 027bb48

Reply:
This kind of check doesn't work for e.g. Xenial. Please use the same approach used elsewhere in this repo: enumate all past distros for which the workaround *shouldn't* be applied.